src/components/e-select.vue

```diff
@@ -151,7 +151,7 @@
     position: relative;
 
     &__select {
-      background: url('../assets/icons/i-arrow--down--info.svg') no-repeat right 5px center;
+      background: url('../assets/icons.svg#arrow--down') no-repeat right 5px center / 22px 22px;
```

Comment:
is there a specific reason, why you are not using the `icon` mixin for all occurrences in components?